frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/index.tsx

```diff
@@ -79,14 +63,59 @@ class Items extends React.Component<
       });
   };
 
+  handleDelete = (id: number) => {
+    axios
+      .delete(window.location.origin + `/core/academix/admin/items/${id}`)
+      .then((result: AxiosResponse<Category[]>) => {
+        if (result.status == 204) {
+          this.fetchItems();
+        }
+      })
+      .catch((error) => {
+        handleApiError(
+          error,
+          'Something went wrong when trying to delete the item'
+        );
+      });
+  };
+
+  columns = [
+    {
+      title: 'Id',
+      dataIndex: 'id',
+      key: 'id',
+    },
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      key: 'edit',
+      // eslint-disable-next-line react/display-name
+      render: (text: string, record: Category) => (
+        <div>
+          <a>Edit</a>
+          <Divider type="vertical" />
+          <Popconfirm
+            title="Sure to delete?"
+            onConfirm={() => this.handleDelete(record.id)}
+          >
+            <a className={mainStyles.delete}>Delete</a>
```

Comment:
Don't use the anchor tag. (anchor tag is using to add hyperlinks)

Reply:
Use button (link typed button) or custom CSS

Reply:
ok I'll remove it.